### PR TITLE
fix(suite-native): account balance inconsistency

### PR DESCRIPTION
**Pull request by PeKne**
Account displayed balance reflects also transactions in the pending state.

### CODE REVIEW

File: suite-common/wallet-core/src/accounts/accountsReducer.ts [hunk 1]
```diff
@@ -385,6 +385,11 @@ export const selectAccountBalance = createMemoizedSelector(
     account => account?.balance ?? null,
 );
 
+export const selectAccountFormattedBalance = createMemoizedSelector(
```
Comment (vytick) [inline suite-common/wallet-core/src/accounts/accountsReducer.ts:388]: Maybe `selectAccountCryptoBalanceWithStaking` would be better or staking balance is not expected here?

Comment (PeKne) [reply suite-common/wallet-core/src/accounts/accountsReducer.ts:388]: I believe that is not expected there.

**Comment by matejkriz** [comment]
Notes for QA (@STew790) :  Test with both bitcoin units (Bitcoin and Satoshis) and make sure to check all the places where account balance is visible 🙏

**Review by vytick — APPROVED** [review]
lgtm

**Comment by STew790** [comment]
**QA NOK**
I can see different value on the accounts modal then in the actual account while the transaction is pending.
After I killed the app and started it again, it showed correctly

https://github.com/user-attachments/assets/7e4075fa-db95-4bb5-985a-b0fa8d195b23

**Info**
24.12.1   a9a2d76

**Pull request merged**